cmd/kubeadm/app/phases/upgrade/compute.go

```diff
@@ -60,7 +60,7 @@ func (u *Upgrade) CanUpgradeEtcd() bool {
 type ClusterState struct {
 	// KubeVersion describes the version of the Kubernetes API Server, Controller Manager, Scheduler and Proxy.
 	KubeVersion string
-	// DNSType
+	// DNSType the DNS add-on to be used.
 	DNSType kubeadmapi.DNSAddOnType
 	// DNSVersion describes the version of the kube-dns images used and manifest version
```

Comment:
please also change this to:
`DNSVersion describes the version of the DNS add-on`

Reply:
has been updated, `DNSVersion` is changed to `DNSType`.

Reply:
could you please also replace this:
> // DNSVersion describes the version of the kube-dns images used and manifest version

with:
> // DNSVersion describes the version of the DNS add-on.

Reply:
YES，updated.